doc/connection-backoff.md

```diff
@@ -24,9 +24,25 @@ ConnectWithBackoff()
          != SUCCESS)
     SleepUntil(current_deadline)
     current_backoff = Min(current_backoff * MULTIPLIER, MAX_BACKOFF)
-    current_deadline = now() + current_backoff
+    current_deadline = now() + current_backoff +
+      UniformRandom(-JITTER * current_backoff, JITTER * current_backoff)
+
 ```
 
+With specific parameters of
+INITIAL_BACKOFF = 20 seconds
```

Comment:
This was rather stupid on my part: I'd already solved this problem correctly with MIN_CONNECT_TIMEOUT.

That is, this algorithm describes an exponential backoff on the *start* of connection attempts, but we (by default at least) don't want to be giving up on a connection attempt as early as 1-2 seconds. That's what MIN_CONNECT_TIMEOUT solves, and setting it to something reasonable lets us have a reasonable INITIAL_BACKOFF.